### PR TITLE
5-FixReactImport and path imports

### PR DESCRIPTION
**Pull request by m0cah**
Allows us to not have to import react every time.

### CODE REVIEW

**Comment by JoshuaVSherman** [comment]
@m0cah here are the following lines of code that are needed in your tsconfig.json file (I am referencing to the JaMmusic one)
line 90 `"include":["src","test"],`
change line 17 to be ` "jsx": "react-jsx",`

File: tsconfig.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
-      "paths": {
-        "*": ["types/*"]
-      },
+    "paths": {
```
Comment (JoshuaVSherman) [inline tsconfig.json:3]: @m0cah this is correct, but you need to also make the changes I mention in my review from last week please

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (m0cah) [reply tsconfig.json:3]: Got it, will do it by tonight

File: tsconfig.json [hunk 2]
```diff
@@ -86,5 +86,6 @@
       /* Advanced Options */
       "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
     },
-    "exclude": ["dist", "node_modules", "test", "webpack.config.js"]
+    "exclude": ["dist", "node_modules", "test", "webpack.config.js"],
```
Comment (JoshuaVSherman) [inline tsconfig.json]: "exclude": ["dist", "node_modules", "webpack.config.js"]

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]
if we are including type-checking for the test files, you need to remove that from the exclude

File: tsconfig.json (hunk 2 above)
Comment (JoshuaVSherman) [reply tsconfig.json]: @m0cah one line is still not correct

Comment (JoshuaVSherman) [reply tsconfig.json]: @m0cah please remove the "test" from the exclude array (we want to include the test files for typechecking), webpack will exclude them when it does the build for prod

**Comment by JoshuaVSherman** [comment]
@m0cah I am still waiting for you to make this one line change I have requested please

File: tsconfig.json (hunk 2 above)
Comment (m0cah) [reply tsconfig.json]: Hey Josh, sorry for the delay. Should be fixed now.

**Pull request merged**